.github/workflows/build.yaml

```
@@ -0,0 +1,7 @@
+name: Build Typst document
+on: push
+
+jobs:
+  build_typst_documents:
```

actionlint (1.7.4)

"steps" section is missing in job "build_typst_documents"

(syntax-check)